packages/components/src/tools-panel/tools-panel/hook.ts

```diff
@@ -48,6 +48,13 @@ const generateMenuItems = ( {
 	return menuItems;
 };
 
+const isMenuItemTypeEmpty = (
+	obj: ToolsPanelMenuItems[ ToolsPanelMenuItemKey ]
+) =>
+	obj &&
+	Object.keys( obj ).length === 0 &&
+	Object.getPrototypeOf( obj ) === Object.prototype;
```

Comment:
Given the typing here for `ToolsPanelMenuItems` do we need the additional check for the object prototype?

Reply:
I was thinking the same thing, seems like the check to ensure we are dealing with an object is redundant?

Reply:
> do we need the additional check for the object prototype?

No, we do not. Good call.